contracts/wawaka/CMakeLists.txt

```diff
@@ -18,10 +18,31 @@ IF(NOT EXISTS $ENV{PDO_HOME})
   MESSAGE(FATAL "PDO_HOME environment variable not defined")
 ENDIF()
 
+IF (NOT DEFINED ENV{WASM_MEM_CONFIG})
+  MESSAGE(FATAL_ERROR "WASM_MEM_CONFIG environment variable not defined!")
+ENDIF()
+
 SET(PDO_TOP_DIR "${CMAKE_CURRENT_SOURCE_DIR}/../../")
 
+SET(WASM_MEM_CONFIG "$ENV{WASM_MEM_CONFIG}")
+
 SET(CONTRACT_EXPORTS "\"['_ww_dispatch', '_ww_initialize']\"")
 
+# Set the memory configuration for emscripten
+IF (WASM_MEM_CONFIG STREQUAL "SMALL")
+  SET(MODULE_STACK_SIZE 4096)
+  SET(MODULE_MAX_MEMORY 65536)
+  message(STATUS "Building contracts for SMALL memory configuration")
+ELSEIF (WASM_MEM_CONFIG STREQUAL "LARGE")
+  SET(MODULE_STACK_SIZE 1048576)
+  SET(MODULE_MAX_MEMORY 4194304)
+  message(STATUS "Building contracts for LARGE memory configuration")
+ELSE()
+  SET(MODULE_STACK_SIZE 32768)
+  SET(MODULE_MAX_MEMORY 524288)
+  message(STATUS "Building contracts for MEDIUM memory configuration")
+ENDIF ()
+
```

Comment:
emscripten allows you to specify more meaningful numbers such as 16KB and 8MB. suggest we use those.

Reply:
Ok, will change these.